api/src/main/resources/liquibase.xml

```diff
@@ -1710,6 +1710,23 @@
             ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
         </sql>
     </changeSet>
+
+    <changeSet id="hospital-core:2022-12-21-18:02" author="vic">
+        <preConditions onFail="MARK_RAN" onError="WARN">
+            <dbms type="mysql"/>
+        </preConditions>
+        <comment>Add table cycle</comment>
+        <sql>
+            CREATE TABLE IF NOT EXISTS `cycle` (
+            `id` int(11) NOT NULL AUTO_INCREMENT,
+            `summary_notes` varchar(255) DEFAULT NULL,
+            `active` tinyint(1) NOT NULL DEFAULT '0',
+            `outcome` varchar(225) DEFAULT NULL,
+            `regimen_id` int(11) DEFAULT NULL,
```

Comment:
Looks like there is no explicit creation of the referential integrity things via code, hope it still works

Reply:
there are alter statements that sort this out